tweet/serializers.py

```diff
@@ -1,13 +1,33 @@
 from rest_framework import serializers
-from tweet.models import Tweet
+from tweet.models import Tweet, User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    # email = serializers.CharField(max_length=100,required=None)
+    # password = serializers.CharField(max_length=100,required=None)
+
+    class Meta:
+        model = User
+        fields = ['id', 'email', 'password', 'created_at', 'updated_at']
+
+    def create(self, validated_data):
+        return User.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.email = validated_data["email"]
+        instance.password = validated_data["password"]
+        instance.save()
+        return instance
+
 
 class TweetSerializer(serializers.ModelSerializer):
-    body = serializers.CharField(max_length=100,required=None)
-    is_retweet = serializers.BooleanField(default=False)
+    # body = serializers.CharField(max_length=100,required=None)
+    # is_retweet = serializers.BooleanField(default=False)
```

Comment:
not serialize ?

Reply:
ตอนแรกพี่ใส่ มัน ตาม guide ใน tutorial แล้วพี่ลองเอาออก มันก็ไม่มีผล เพราะ ก็ serialize ได้เหมือนกันเลยเอาออก